http-server-netty/src/main/java/io/micronaut/http/server/netty/NettyHttpRequest.java

```diff
@@ -432,7 +391,48 @@ public void release() {
             formRouteCompleter.release();
         }
         if (attributes != null) {
-            attributes.forEach(CLEANUP_ATTRIBUTES);
+            attributes.forEach(cleanup());
```

Comment:
no it needs to be a method ref, something like NettyHttpRequest::cleanup

Reply:
Actually, looking at LambdaMetafactory, I think this change is altogether unnecessary: You only need to make releaseIfNecessary static, and the lambda won't capture anything, and will be cached.